src/modules/songs/forms/CreateResourceForm/CreateResourceForm.tsx

```
@@ -0,0 +1,155 @@
+      url: "",
+    },
+    mode: "onBlur",
+    reValidateMode: "onBlur",
+  });
+
+  if (!userId) {
+    return null;
+  }
+
+  // TODO: handle if there is no user data (assume this will be handled before it reaches here though?)
+  const { data: userData } = trpcApi.user.getUser.useQuery({
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

Hooks should not be called after an early return.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)